operator-framework-core/src/test/java/io/javaoperatorsdk/operator/sample/simple/TestCustomResource.java

```diff
@@ -2,10 +2,12 @@
 
 import io.fabric8.kubernetes.client.CustomResource;
 import io.fabric8.kubernetes.model.annotation.Group;
+import io.fabric8.kubernetes.model.annotation.Kind;
 import io.fabric8.kubernetes.model.annotation.Version;
 
 @Group("sample.javaoperatorsdk.io")
 @Version("v1")
+@Kind("TestCustomResource")
```

Comment:
Why is this needed?

Reply:
This is needed to compare resources when we register a new reonciler. There was a tests for the registration of the same resource with multiple versions - what should fail. It did not fail without this, could not determine the kind. It was not failing until now because there were junit4 annotations used on test class

Reply:
see `ControllerManagerTest.java` changes

Reply:
If that functionality should work without this annotation, that should be a separate issues IMHO.

Reply:
It shouldn't be needed here. Just for the v2.